### DISCUSSION TITLE
Add arguments in the linkout function (jexl)

### DISCUSSION THREAD

**Discussion by Chi-HsienChang** [GMOD/jbrowse-components#3635] [Q&A] [answered]
Hi JBrowse2 developers,

I want to ask: 

It is possible to add arguments in the linkout function, allowing users to decide the linkout destination by themselves.

Here's I currently try do it as follows:

![image](https://user-images.githubusercontent.com/114167559/231227191-f868338e-91b8-40ae-96bf-f734444917fc.png)

After that, the Feature details turn to empty QQ.
![image](https://user-images.githubusercontent.com/114167559/231228254-16403924-9651-4353-9921-09d1cd70749f.png)

Thank you so much!

**Comment by cmdcolin** [comment, marked as answer]
What you describe should work, e.g. you can provide extra params like that. Are there any messages in the javascript console?

**Reply by Chi-HsienChang** [reply]
Hi @cmdcolin, 

There are no messages in the javascript console.  However, the Feature details (.gff) turn to empty in JBrowse 2.

Thank you!

**Reply by Chi-HsienChang** [reply]
Hi @cmdcolin,

I find my bug, I forget to modify the subfeatures!

Sorry!

Thank you!
